<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.5720" tests="2" failures="1">
  <testsuite name="Root Suite" timestamp="2021-12-30T20:50:45" tests="0" file="cypress/integration/dashboard.feature" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Dashboard" timestamp="2021-12-30T20:50:45" tests="2" time="0.5720" failures="1">
    <testcase name="Dashboard I can see the recipe dashboard" time="0.5720" classname="I can see the recipe dashboard">
    </testcase>
    <testcase name="Dashboard I can see the recipe card" time="0.0000" classname="I can see the recipe card">
      <failure message="Timed out retrying after 4000ms: Expected to find content: &apos;View Recipe&apos; within the element: &lt;body&gt; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: 'View Recipe' within the element: <body> but never did.
    at Context.eval (http://localhost:3000/__cypress/tests?p=cypress/integration/dashboard.feature:13:34)
    at Context.resolveAndRunStepDefinition (http://localhost:3000/__cypress/tests?p=cypress/integration/dashboard.feature:12306:29)
    at Context.eval (http://localhost:3000/__cypress/tests?p=cypress/integration/dashboard.feature:11627:35)]]></failure>
    </testcase>
  </testsuite>
</testsuites>